CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+### Changed (let op: release wk50)
```

Comment:
De opmerking klopt hier niet. Daarbij, merge master hier in dan kan je de changelog op de juiste plek zetten

Reply:
aangepast